newsroom/template_filters.py

```diff
@@ -15,8 +15,7 @@
 
 def to_json(value):
     """Jinja filter to address the encoding of special values to json"""
-
-    return bson_json_util.dumps(value)
+    return SuperdeskJSONEncoder().dumps(value)
```

Comment:
It might be better to use `app.json_encoder().dumps(value)` so that this code will use whatever is registered when bootstrapping the app (in case the encoder changes, for whatever reason).

Reply:
Done!